data.js

```diff
@@ -0,0 +1,163 @@
+  modaldialog.append(header);
+  const desc = document.createElement('div');
+  desc.setAttribute('class', 'project-name');
+  header.append(desc);
+  const h2 = document.createElement('h2');
+  h2.setAttribute('class', 'header-title');
+  h2.innerHTML = `${project.name}`;
+  desc.append(h2);
+  const ul = document.createElement('ul');
+  for (let j = 0; j < project.languages.length;) {
+    const li = document.createElement('li');
+    const a1 = document.createElement('a');
+    a1.setAttribute('class', 'read-text');
+    a1.innerHTML = project.languages[j];
+    li.append(a1);
+    ul.append(li);
+    // eslint-disable-next-line no-plusplus
+    j++;
+  }
+
+  desc.append(ul);
+  header.append(desc);
+  const closebtn = document.createElement('button');
+  closebtn.setAttribute('class', 'close-modal');
+  closebtn.setAttribute('data-close', 'close');
+  closebtn.innerHTML = '&times;';
+  header.append(closebtn);
+  const mainsection = document.createElement('section');
+  mainsection.setAttribute('class', 'mains');
+  mainsection.setAttribute('border', '2px solid gray');
+  modaldialog.append(mainsection);
+  const leftDiv = document.createElement('div');
+  leftDiv.setAttribute('class', 'left-div');
+  const projectImage = document.createElement('img');
+  projectImage.setAttribute('class', 'project-img');
+  projectImage.setAttribute('src', project.projectimage);
+  leftDiv.append(projectImage);
+  mainsection.append(leftDiv);
+  const rightDiv = document.createElement('div');
+  rightDiv.setAttribute('class', 'right-div');
+  mainsection.append(rightDiv);
+  const p = document.createElement('p');
+  p.setAttribute('class', 'project-details');
+  p.innerHTML = project.projectdetails;
+  rightDiv.append(p);
+  const bottomDiv = document.createElement('div');
+  bottomDiv.setAttribute('class', 'live-section');
+  const btn1 = document.createElement('button');
+  btn1.setAttribute('class', 'div-1--right-btn');
+  btn1.innerHTML = project.liveVersion;
+  bottomDiv.append(btn1);
+  const btn2 = document.createElement('button');
+  btn2.setAttribute('class', 'div-1--right-btn');
+  btn2.innerHTML = project.sourceLink;
+  bottomDiv.append(btn2);
+  rightDiv.append(bottomDiv);
+};
+
+const displayProject = document.querySelectorAll('[data-open]');
+
+for (const btn of displayProject) {
+  btn.addEventListener('click', (e) => {
+    // eslint-disable-next-line spaced-comment
+    //document.querySelector('.modal').classList.add(isVisible);
+    e.stopPropagation();
+    projects.forEach((project) => createDialog(project));
+  });
+}
+
+const hideProject = document.querySelectorAll('[close]');
+
+for (const btn of hideProject) {
+  btn.addEventListener('click', (e) => {
+    e.stopPropagation();
+    document.querySelector('.modal').classList.remove(isVisible);
+  });
+}
+
+document.addEventListener('click', (e) => {
+  if (e.target === document.querySelector('.modal.is-visible'));
+  document.querySelector('.modal').classList.remove(isVisible);
+});
```

Comment:
- Please try fixing the models for the other projects, as only the first project popup shows, I keep getting the error ` Cannot read properties of null (reading 'classList')` on this line, meaning the model you are trying to access is can access the property of classList. 
  The first model works perfectly, so logically, but others don't. If you find this feedback confusing please reach out to me on Slack `@kanu mike chibundu`

- Note: I must commend your styling of the initial popup, you did well, happy coding :+1:

Reply:
Done as advised

Reply:
> Done as advised

Hi @r-ahmed2022 , I'm currently reviewing your project and only the first project has a modal pop up "See Project" button. Can you  please kindly record a screencast to show a demo of your modals because I can only see one "See Project" button